ui/src/admin/actions/index.js

```diff
@@ -104,3 +108,21 @@ export const deleteUserAsync = (user, addFlashMessage) => (dispatch) => {
   // delete user on server
   deleteUserAJAX(user.links.self, addFlashMessage, user.name)
 }
+
+export const updateRoleUsersAsync = (users, role) => async (dispatch) => {
```

Comment:
switch order of arguments to follow convention in the rest of codebase. and actually, the other convention is just to pass in the url directly. let's choose one or the other and stick with it.